oshdb/src/test/java/org/heigit/ohsome/oshdb/osh/OSHRelationTest.java

```diff
@@ -258,4 +258,34 @@ public void testToString() throws IOException {
     assertEquals(expResult, result);
   }
 
+  @Test
+  public void testIssue325() throws IOException {
+    OSHNode hnode1 = OSHNodeImpl.build(new ArrayList<>(List.of(
+        new OSMNode(123L, 2, new OSHDBTimestamp(2L), 2L, 0, new int[]{}, 0, 0),
+        new OSMNode(123L, 1, new OSHDBTimestamp(1L), 1L, 0, new int[]{}, 0, 0))));
+    OSHNode hnode2 = OSHNodeImpl.build(new ArrayList<>(List.of(
+        new OSMNode(124L, 2, new OSHDBTimestamp(2L), 2L, 0, new int[]{}, 0, 0),
+        new OSMNode(124L, 1, new OSHDBTimestamp(1L), 1L, 0, new int[]{}, 0, 0))));
+
+    OSHWay hway1 = OSHWayImpl.build(new ArrayList<>(List.of(
+        new OSMWay(1, 1, new OSHDBTimestamp(1L), 1L, 0, new int[]{}, new OSMMember[]{
+            new OSMMember(123, OSMType.NODE, 0),
```

Comment:
I think it's not an issue, but you use `123` here and `123L` above. The same syntax would look a little bit nicer. Same in the next line.